Project7.md

```diff
@@ -24,7 +24,12 @@ A pseudocode map-reduce style algorithm for computing co-views is something like
 Implementation
 ==============
 
-Spark setup
+Producing the Access Log
+------------------------
+
+The access log that your Spark job will use should be in the form of a file with two columns of values where each row consists of a user-id and an item-id representing an item page view by a logged in user. Similar to project 5's implementation, every time a page view occurs you will need to push the two relevant values to Kafka from the experience layer and have another batch container/script consume the page-view and append it to the aforementioned running log file. 
```

Comment:
Need to reference the fact that somehow the log file has to be written by one container (running the new batch job which reads from Kafka and writes to the file) and the Spark containers that read from the file. Perhaps give a hint as to how to use Docker to do this without actually specifying the exact commands to use.